antarest/launcher/adapters/slurm_launcher/slurm_launcher.py

```diff
@@ -144,18 +144,17 @@ def apply_adequacy_patch(self, launcher_params: LauncherParametersDTO) -> None:
 class SlurmLauncher(AbstractLauncher):
     def __init__(
         self,
-        config: Config,
+        config: SlurmConfig,
         callbacks: LauncherCallbacks,
         event_bus: IEventBus,
         cache: ICache,
         use_private_workspace: bool = True,
         retrieve_existing_jobs: bool = False,
     ) -> None:
+        if not isinstance(config, SlurmConfig):
```

Comment:
We don't really need this now, this will be guaranteed by the type checker in CI